### PR TITLE
Fix #6596: Convert string to plurals

### LINKED ISSUE #6596: String fav_points_remaining needs plurals
Taken over from a comment of @kumy in Crowdin:

The string fav_points_remaining needs plurals to be correctly translated.

### PR DESCRIPTION
**Pull request by UniQP**
Also remove the quotes around the string.

I did not managed to test it properly, since I did not find the place where the string is used.

### CODE REVIEW

**Comment by pstorch** [comment]
I got a build error and Jenkins reported it, too. But now it is green?!?

Build fails:
```
22:05:44  :main:compileBasicDebugJavaWithJavacwarning: A class activity declared in the AndroidManifest.xml cannot be found in the compile path: [android.support.v7.widget.TestActivity]
22:05:55  /home/jenkins/slave/workspace/cgeo pull request/main/src/cgeo/geocaching/log/LogCacheActivity.java:395: error: cannot find symbol
22:05:55                  favCheck.setText(getQuantityString(R.string.fav_points_remaining, premFavPoints, premFavPoints));
```

**Comment by UniQP** [comment]
I fixed the compile error.

**Comment by pstorch** [comment]
Got an Exception:

```
06-07 22:53:19.665 16542-16542/cgeo.geocaching E/AndroidRuntime: FATAL EXCEPTION: main
                                                                 Process: cgeo.geocaching, PID: 16542
                                                                 android.content.res.Resources$NotFoundException: Plural resource ID #0x7f0702a7 quantity=48 item=other
                                                                     at android.content.res.ResourcesImpl.getQuantityText(ResourcesImpl.java:268)
                                                                     at android.content.res.Resources.getQuantityText(Resources.java:362)
                                                                     at android.content.res.Resources.getQuantityString(Resources.java:438)
                                                                     at cgeo.geocaching.log.LogCacheActivity.initializeFavoriteCheck(LogCacheActivity.java:395)
                                                                     at cgeo.geocaching.log.LogCacheActivity.onLoadFinished(LogCacheActivity.java:141)
                                                                     at cgeo.geocaching.connector.gc.GCLoggingManager.onLoadFinished(GCLoggingManager.java:86)
```

File: main/src/cgeo/geocaching/log/LogCacheActivity.java [hunk 1]
```diff
@@ -392,7 +392,7 @@ private void initializeFavoriteCheck() {
         if (ConnectorFactory.getConnector(cache).supportsAddToFavorite(cache, typeSelected)) {
             if (premFavPoints > 0) {
                 favCheck.setVisibility(View.VISIBLE);
-                favCheck.setText(getString(R.string.fav_points_remaining, premFavPoints));
+                favCheck.setText(res.getQuantityString(R.string.fav_points_remaining, premFavPoints, premFavPoints));
```
Comment (pstorch) [inline main/src/cgeo/geocaching/log/LogCacheActivity.java]: I think instead of `R.string` in needs to be `R.plurals`.

**Comment by pstorch** [comment]
BTW: you can test it via Logging a Cache as found it. This text should appear below the log textbox.

**Comment by UniQP** [comment]
Ok, seems to work now.

@pstorch Thanks for your help.

**Comment by pstorch** [comment]
@UniQP thanks :+1:

**Comment by kumy** [comment]
Thanks :+1:

**Pull request merged**